dvc/utils/__init__.py

```diff
@@ -381,7 +381,7 @@ def _visual_center(line, width):
 
 def relpath(path, start=os.curdir):
     path = fspath_py35(path)
-    start = fspath_py35(os.path.abspath(start))
+    start = os.path.abspath(fspath_py35(start))
```

Comment:
We have a test for relpath in `tests/unit/utils/test_fs.py`, need to understand why it didn't catch this.

Reply:
Ah, because it was using strings and not PathInfos. Need to adjust it to test for that too.

Reply:
`relpath()` has very tricky API, which depends on OS and python version, i.e. it may return `PathInfo` or `str` depending on a number of conditions.